modules/webbrowser/include/modules/webbrowser/properties/templatepropertywidgetcef.h

```diff
@@ -66,12 +92,31 @@ class TemplatePropertyWidgetCEF : public PropertyWidgetCEF {
         const std::string& requestString = request;
         auto j = json::parse(requestString);
         try {
-            T value = j.at("value").get<T>();
+            auto command = j.at("command").get<std::string>();
             auto p = static_cast<TemplateProperty<T>*>(getProperty());
-            p->setInitiatingWidget(this);
-            p->set(value);
-            p->clearInitiatingWidget();
-            callback->Success("");
+            if (command == "property.set") {
```

Comment:
These available "commands" should be documented somewhere.

Reply:
What about the set functions which take more than one element?

Reply:
More than one element is resolved. Documentation will have to be added.

Reply:
Added documentation